src/Halogen/Aff/Driver.purs

```diff
@@ -93,8 +93,8 @@ import Halogen.Subscription as HS
 -- | The `dispose` function is called when the top level component is disposed of
 -- | via `HalogenIO`.
 type RenderSpec r =
-  { render
-      :: forall s act ps o
+  { render ::
+      forall s act ps o
```

Comment:
This is a bit odd (the `::` being pulled up a line) -- thoughts on this, @natefaubion?

Reply:
It's debatable, but it's currently consistent with the value level syntax for fields.